pilot/pkg/networking/core/v1alpha3/cluster_builder.go

```diff
@@ -611,11 +613,11 @@ func (cb *ClusterBuilder) buildInboundPassthroughClusters() []*cluster.Cluster {
 	// to support the Dual Stack via Envoy bindconfig, and belows are related issue and PR in Envoy:
 	// https://github.com/envoyproxy/envoy/issues/9811
 	// https://github.com/envoyproxy/envoy/pull/22639
-	if features.EnableDualStack {
-		inboundPassthroughClusterForDS := cb.buildDefaultPassthroughCluster()
-		inboundPassthroughClusterForDS.Name = util.InboundPassthroughClusterDualStack
-		inboundPassthroughClusterForDS.Filters = nil
-		inboundPassthroughClusterForDS.UpstreamBindConfig = &core.BindConfig{
+	if features.EnableDualStack && cb.supportsIPv4 && cb.supportsIPv6 {
```

Comment:
suggest wrap up as a function

Reply:
okay, that's okay, and BTW, I think there should not be merged into one cluster for now.

Reply:
Hi @hzxuzhonghu, done!